lib/components/user/monitored-trip/trip-status.js

```diff
@@ -0,0 +1,156 @@
+import {
+  getDateFormat,
+  getLongDateFormat,
+  getTimeFormat
+} from '@opentripplanner/core-utils/lib/time'
+import AlertsBody from '@opentripplanner/itinerary-body/lib/TransitLegBody/alerts-body'
+import React from 'react'
+import { Button, Glyphicon, Panel } from 'react-bootstrap'
+import { connect } from 'react-redux'
+import styled from 'styled-components'
+
+import {
+  confirmAndDeleteUserMonitoredTrip,
+  planNewTripFromMonitoredTrip,
+  togglePauseTrip,
+  toggleSnoozeTrip
+} from '../../../actions/user'
+import getRenderData from './trip-status-rendering-strategies'
+
+const StyledFooterButton = styled(Button)`
+  margin-right: 10px;
+`
+
+export function FooterButton ({ bsStyle, glyphIcon, onClick, text }) {
+  return (
+    <StyledFooterButton bsStyle={bsStyle} onClick={onClick}>
+      <><Glyphicon glyph={glyphIcon} /> {text}</>
+    </StyledFooterButton>
+  )
+}
+
+function MonitoredTripAlerts ({ alerts, longDateFormat, timeFormat }) {
+  let alertsHeader
+  if (alerts.length === 1) {
+    alertsHeader = `${alerts.length} alert!`
+  } else {
+    alertsHeader = `${alerts.length} alerts!`
+  }
+
+  return (
+    <Panel.Body>
+      <h4>{alertsHeader}</h4>
+      <AlertsBody
+        alerts={alerts}
+        longDateFormat={longDateFormat}
+        timeFormat={timeFormat}
+      />
+    </Panel.Body>
+  )
+}
+
+/**
+ * A functional component that is used to display summary information about a
+ * certain monitored trip. It is expected that the renderData for the trip is
+ * passed to this component with the necessary data for rendering each portion
+ * of this component.
+ */
+function TripStatus ({
+  confirmAndDeleteUserMonitoredTrip,
+  longDateFormat,
+  planNewTripFromMonitoredTrip,
+  renderData,
+  timeFormat,
+  togglePauseTrip,
+  toggleSnoozeTrip
+}) {
+  return (
+    <Panel bsStyle={renderData.panelBsStyle}>
+      {/* Heading */}
+      <Panel.Heading>
+        <h3>{renderData.headingText}</h3>
+        <h6>{renderData.lastCheckedText}</h6>
```

Comment:
Sorry I'm doubling down on this since it is not a heading... `<div>` and/or `<small>`?

Reply:
Changed to `<small>` in 8f071e25ea62586b9d67fbef14430316d2ae2abe